contrib/mutex_owner.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env drgn
+# Copyright (c) Cloudflare
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""List the task holding a mutex and which tasks are waiting."""
+
+import sys
+import drgn
+from drgn import FaultError, NULL, Object, cast, container_of, execscript, offsetof, reinterpret, sizeof, stack_trace
+from drgn.helpers.common import *
+from drgn.helpers.linux import *
+
+def comm_to_str(comm):
+	return ''.join(chr(x) for x in comm if x > 0)
+
+def main(mutex_name):
+	print("Mutex details for '%s'" % mutex_name)
+	print("="*40)
+
+	m = prog[mutex_name]
+	addr = cast("unsigned long", m.owner.counter) & ~0x7
+	if addr != 0:
+		t = cast("struct task_struct *", addr)
+		owner = "{}-{}".format(comm_to_str(t.comm.value_()), int(t.pid))
```

Comment:
The typical way to get a string from a character array is:

```suggestion
		owner = "{}-{}".format(t.comm.string_().decode(), int(t.pid))
```

Same thing below, then you can delete `comm_to_str`.